x/pylons/keeper/grpc_query_google_iap_order.go

```diff
@@ -21,6 +22,7 @@ func (k Keeper) GoogleInAppPurchaseOrder(c context.Context, req *types.QueryGetG
 	ctx := sdk.UnwrapSDKContext(c)
 
 	if !k.HasGoogleIAPOrder(ctx, req.PurchaseToken) {
+		fmt.Println("The Key was not found")
```

Comment:
Is this fmt.Println needed at this point? Also, is it effective, is the fmt import necessary? shouldnt we be using something else like assertions